pathfinder/tests/fixtures/core.py

```diff
@@ -1,73 +1,73 @@
 import random
-from typing import List, Dict, Tuple
+from typing import List
 from unittest.mock import Mock
 
 import pytest
-from _pytest import monkeypatch
 from eth_utils import to_checksum_address, encode_hex, keccak
 from raiden_libs.contracts import ContractManager
 from web3 import Web3
+from web3.contract import Contract
 
-from pathfinder.contract.token_network_contract import TokenNetworkContract
 from pathfinder.pathfinding_service import PathfindingService
 from pathfinder.token_network import TokenNetwork
-from pathfinder.utils.types import ChannelId, Address
 
 
 @pytest.fixture
 def populate_token_networks(
     token_networks: List[TokenNetwork],
-    token_network_contracts: List[TokenNetworkContract],
-    monkeypatch: monkeypatch
+    token_network_contracts: List[Contract],
 ) -> None:
 
-    def get_channel_deposits_patched(
-        self: TokenNetworkContract,
-        channel_id: ChannelId
-    )-> Dict[Address, int]:
+    for token_network in token_networks:
+        # some random magic to add channels
+        for channel_id in range(10):
```

Comment:
This is why it doesn't find any paths. You're only creating 10 channels over an address space of 100 different addresses (line 25). Set this to 100, same in line 37 (to add deposits to all channels), and reduce the number of nodes to something like 10 (line 25).

Apart from that I see Lefteris' issues with using `random.seed()` multiple times but this should be fixed in #17 once it's rebased on this PR. #17 pregenerates a set of private keys and addresses that are reused, so no need to generate them procedurally anymore.

So I'd say leave the `random.seed()` stuff in for now and we will fix it with #17.